executor/services/api_client.py

```
@@ -540,3 +540,62 @@ def fetch_task_attachments(task_id: str, auth_token: str) -> TaskAttachmentsResu
+                f"bot_name={bot_name}"
+                f"response={response}"
+            )
+            return data
+        else:
+            logger.warning(
+                f"[fetch_task_execution_info] API returned no response for task {task_id}"
+            )
+            return None
+
+    except Exception as e:
+        logger.exception(f"[fetch_task_execution_info] Error: {e}")
```

Ruff (0.15.2)

[warning] Redundant exception object included in `logging.exception` call

(TRY401)